dbms/src/Flash/tests/gtest_window_executor.cpp

```diff
@@ -438,7 +421,7 @@ try
                   .window(Lead2(concat(col("value"), concat(lit(Field(String("0"))), col("value"))), lit(Field(static_cast<UInt64>(1)))), {"order", false}, {"partition", false}, MockWindowFrame())
                   .build(context);
     result.emplace_back(toNullableVec<String>({"b0b", "c0c", "d0d", {}, "f0f", "g0g", "h0h", {}}));
-    executeWithConcurrency(request, result);
+    executeAndAssertColumnsEqual(request, result);
```

Comment:
How can we print the line number in gtest_xxx.cpp when test failed？

Reply:
good point

Reply:
now the fail msg will be
```
Value of: assert_func(executeStreams(request, concurrency))
  Actual: false (
Columns rows mismatch
expected_rows: 0
actual_rows: 1)
Expected: true
test info:
    file: /data1/huanghaisheng/git-repo/tiflash/dbms/src/Flash/tests/gtest_limit_executor.cpp
    line: 50
    test_case_name: ExecutorLimitTestRunner
    test_func_name: Limit
    enable_planner: false
    concurrency: 1
    block_size: 1
    dag_request: 
limit_1 | 1
 table_scan_0 | {<0, String>}
```